src/BookingRow.js

```diff
@@ -1,13 +1,27 @@
-import React from "react";
+import React, { useState } from "react";
 import moment from "moment";
 
 function calcDeff(chkIn, chkOut) {
   return moment(chkOut, "YYYY-MM-DD").diff(moment(chkIn, "YYYY-MM-DD"), "days");
 }
 
 const BookingRow = props => {
+  const [bookingRowClass, setBookingRowClass] = useState("");
+  const [selected, setSelected] = useState(false);
```

Comment:
Really excellent work! Well done 🙂 Can you explain how this solves the problem in exercise 16?